scripts/install/install_macos.sh

```diff
@@ -272,6 +278,35 @@ check_prereqs()
   decrease_indent
 }
 
+# Test connection to BindPlane if it was specified
+connection_check()
+{
+  if [ "$check_bp_url" = "true" ] ; then
+    if [ -n "$opamp_endpoint" ]; then
+      HTTP_ENDPOINT="$(echo "${opamp_endpoint}" | sed 's#^ws#http#' | sed 's#/v1/opamp$##')"
+      info "Testing connection to BindPlane: $fg_magenta$HTTP_ENDPOINT$reset..."
+
+      if curl -s "${HTTP_ENDPOINT}" > /dev/null; then
```

Comment:
I tested this and noticed curl took a long time to timeout when I entered a bad hostname.
I think we should set the timeout, maybe 10s is too short but something shorter like this.
```suggestion
      if curl --max-time 10 -s "${HTTP_ENDPOINT}" > /dev/null; then
```

Reply:
Good idea. I didn't even consider this, because both my Mac and my Linux test system time out very quickly.

I think 20 seconds is a reasonable amount of time to wait.